incubator/neo4j/templates/secrets.yml

```diff
@@ -0,0 +1,18 @@
+{{- if .Values.authEnabled -}}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: app: {{ template "neo4j.name" . }}
```

Comment:
You should have only changed the `app` label, not the name. Please change it back to `{{ template "neo4j.secrets.fullname" . }}`